install.sh

```diff
@@ -85,13 +90,59 @@ pacman --noconfirm -S \
 	grub \
 	steam
 
-# install nvidia graphics if needed
-if lspci | grep -E -i '(vga|3d)' | grep -i nvidia > /dev/null; then
-	pacman --noconfirm -S nvidia nvidia-utils lib32-nvidia-utils
-fi
-
 systemctl enable NetworkManager lightdm bluetooth
 
+# gpu detection/driver installation
+elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon, amdgpu' > /dev/null; then
+	echo "AMD GPU with radeon kernel detected, switching kernel module and installing drivers..."
+	echo "blacklist radeon" > /etc/modprobe.d/blacklist.conf
```

Comment:
perhaps combine `blacklist.conf` and `amdgpu.conf` into one `gameros.conf`?

Reply:
If we're going to do this, it'd be a good idea to pull this off at a later stage.

Reply:
why?